clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -97,6 +99,8 @@ class AutoScalingWorker {
   private List<AmazonBlockDevice> blockDevices
   private Map<String, String> tags
   private List<AmazonAsgLifecycleHook> lifecycleHooks
+  private Boolean setLaunchTemplate
```

Comment:
I just want to confirm that this flag is being managed by config, and not something that is expected from a user's create request.

Reply:
So this here is request based to allow falling back to launch configs.
Your comment reminded me that this should default to true (overrideable per request).

But to your point, we should have a configurable default that would be flipped to true to enable the functionality for everyone and use this flag for overrides,
I will add that to this PR.

Reply:
Actually, let me do that in a follow-up PR.

Reply:
ok sounds good, that was my main concern here. 

Sorry I should have been a bit more specific here. We talked about config being managed in one place for defaulting. This means that UI requests will not have this flag, and rely on the default value. API requests can override using `setLaunchTemplate`.